runtime/jvmti/jvmtiThread.c

```diff
@@ -415,12 +415,14 @@ jvmtiInterruptThread(jvmtiEnv* env,
 
 		rc = getVMThread(currentThread, thread, &targetThread, FALSE, TRUE);
 		if (rc == JVMTI_ERROR_NONE) {
-			omrthread_interrupt(targetThread->osThread);
+			if (NULL != targetThread) {
```

Comment:
Shouldn't this check be inside `ifdef` of Java 19 ?

Reply:
Updated.